docs/src/equations.md

```diff
@@ -27,7 +27,10 @@ We make use of the following operators
 
 * ``I^c`` is the [face-to-center reconstruction operator](https://clima.github.io/ClimaCore.jl/stable/operators/#ClimaCore.Operators.InterpolateF2C)
 * ``I^f`` is the [center-to-face reconstruction operator](https://clima.github.io/ClimaCore.jl/stable/operators/#ClimaCore.Operators.InterpolateC2F)
-  - Currently this is just the arithmetic mean, but we will need to use a weighted version with stretched vertical grids.
+  - Currently this is just the arithmetic mean, for uniform grids. 
+* ``WI^f`` is the [center-to-face weighted reconstruction operator](https://clima.github.io/ClimaCore.jl/stable/operators/#ClimaCore.Operators.WeightedInterpolateC2F)
+  - This defines the generalised weighted interpolation operator, for use with stretched grids. 
```

Comment:
```suggestion
  - This defines the weighted interpolation operator, for stretched grids. 
```

Reply:
Add a definition of how the weighting works, so we can read this as a self-contained documentation. I.e., write ``WI^f(J, \rho) = .... `, so readers can understand the function signature.